src/main.rs

```diff
@@ -0,0 +1,130 @@
+#![feature(try_blocks)]
```

Comment:
I don't see them used in code

Reply:
Yeah, I used them initially, but then replaced with regular iterators

Reply:
Then remove it maybe?